autoload/lookup.vim

```diff
@@ -44,7 +44,7 @@ endfunction
 
 " s:find_local_func_def() {{{1
 function! s:find_local_func_def(funcname) abort
-  if search('\c\v<fu%[nction]!?\s+%(s:|\<sid\>)\zs\V'.a:funcname.'\>', 'bsw') != 0
+  if search('\c\v<fu%[nction]!?\s+%(s:|\<sid\>|\<SID\>)\zs\V'.a:funcname.'\>', 'bsw') != 0
```

Comment:
But `S:` etc. would be invalid syntax anyway. The problem is that this has to work as well:

```vim
function! <sId>f()
endfunction

call <SiD>f()
```